server/src/logger/logger.go

```diff
@@ -46,6 +47,7 @@ func FromRequest(r *http.Request) *zap.SugaredLogger {
 // to the pre initialized logger.
 func FromContext(ctx context.Context) *zap.SugaredLogger {
 	l, ok := ctx.Value(ctxRequestLogger).(*zap.SugaredLogger)
+	l.WithOptions()
```

Comment:
Why are you applying empty options?

Reply:
You're right, that line makes no sense.
To be honest, I didn't even realize I added that line.
Will remove it!